src/test/java/seedu/address/model/person/NameContainsKeywordsPredicateTest.java

```diff
@@ -70,7 +70,7 @@ public void test_nameDoesNotContainKeywords_returnsFalse() {
 
         // Keywords match phone, email and address, but does not match name
         predicate = new NameContainsKeywordsPredicate(Arrays.asList("12345", "alice@email.com", "Main", "Street"));
-        assertFalse(predicate.test(new PersonBuilder().withName("Alice").withPhone("12345")
+        assertFalse(predicate.test(new PersonBuilder().withName("Alice").withPhone("999")
```

Comment:
Yeah this test case was the problem that I was referring to. 999/666/888 numbers